specs/swagger.yml

```diff
@@ -981,14 +981,248 @@ paths:
+        404:
+          description: Not Found
+  '/user/profile':
+    get:
+      tags:
+        - User
+      description: Get current user profile data
+      operationId: getUserProfile
+      parameters:
+        - in: header
+          name: Authorization
+          description: User token
+          required: true
+          type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/UserProfileDetailsResponse'
+        401:
+          description: Not Authorized
+        404:
+          description: Not Found
+    patch:
+      tags:
+        - User
+      description: Update user profile
+      operationId: updateUserProfile
+      parameters:
+        - in: header
+          name: Authorization
+          description: User Token
+          required: true
+          type: string
+        - in: body
+          name: profileData
+          description: Updated profile data
+          required: true
+          schema:
+            $ref: '#/definitions/UserProfileUpdatesRequest'
+      responses:
+        200:
+          description: Updated user profile
+          schema:
+            $ref: '#/definitions/UserProfileDetailsResponse'
+          headers:
+            X-Timestamp:
+              type: number
+              description: FogController server timestamp
+        401:
+          description: Not Authorized
+        404:
+          description: Not Found
+    delete:
+      tags:
+        - User
+      description: Delete account
+      operationId: deleteAccount
+      parameters:
+        - in: header
+          name: Authorization
+          description: User Token
+          required: true
+          type: string
+      responses:
+        204:
+          description: Success
+          headers:
+            X-Timestamp:
+              type: number
+              description: FogController server timestamp
+        401:
+          description: Not Authorized
+        404:
+          description: Not Found
+  '/user/password':
+    patch:
+      tags:
+        - User
+      description: change password
+      operationId: changePassword
+      parameters:
+        - in: header
+          name: Authorization
+          description: User Token
+          required: true
+          type: string
+        - in: body
+          name: passwordUpdates
+          description: current and new password
+          required: true
+          schema:
+            $ref: '#/definitions/PasswordChangeRequest'
+      responses:
+        204:
+          description: Success
+          headers:
+            X-Timestamp:
+              type: number
+              description: FogController server timestamp
+        401:
+          description: Not Authorized
+        404:
+          description: Not Found
+    delete:
+      tags:
+        - User
+      description: Reset password
+      operationId: resetPassword
+      parameters:
+        - in: header
```

Comment:
`Authorization` is not required... users request to reset password if they are not able to login, so there is not any accessToken at this point 😄 
email must be passed as query parameter, instead